src/components/Property/RepairsFinder/RepairsFinderForm.tsx

```diff
@@ -0,0 +1,243 @@
+          ? 2
+          : daysInHours(priority.daysToComplete),
+    })
+
+    // follow-on parent
+    const parentWorkOrderId =
+      formData?.isFollowOn === 'true' && formData?.parentWorkOrder
+        ? formData.parentWorkOrder
+        : null
+
+    onFormSubmit(scheduleWorkOrderFormData, parentWorkOrderId)
+  }
+
+  const getPropertyInfoOnLegalDisrepair = (propertyReference) => {
+    frontEndApiRequest({
+      method: 'get',
+      path: `/api/properties/legalDisrepair/${propertyReference}`,
+    })
+      .then((isInLegalDisrepair) =>
+        setIsInLegalDisrepair(isInLegalDisrepair.propertyIsInLegalDisrepair)
+      )
+      .catch((error) => {
+        console.error('Error loading legal disrepair status:', error.response)
+        setLegalDisRepairError(
+          `Error loading legal disrepair status: ${error.response?.status} with message: ${error.response?.data?.message}`
+        )
+      })
+      .finally(() => setLoading(false))
+  }
+
+  useEffect(() => {
+    setLoading(true)
+
+    getPropertyInfoOnLegalDisrepair(propertyReference)
+  }, [])
+
+  return (
+    <>
+      <BackButton />
+      <div className="govuk-grid-row">
+        <div className="govuk-grid-column-two-thirds">
+          <span className="govuk-caption-l lbh-caption">New repair</span>
+          <h1 className="lbh-heading-h1 govuk-!-margin-bottom-2">
+            {hierarchyType?.subTypeDescription}: {address?.addressLine}
+          </h1>
+
+          {loading && <Spinner />}
+
+          {isInLegalDisrepair && !loading && (
+            <WarningInfoBox
+              header="This property is currently under legal disrepair"
+              text="Before raising a work order you must call the Legal Disrepair Team"
+            />
+          )}
+
+          {legalDisrepairError && <ErrorMessage label={legalDisrepairError} />}
+
+          <div className="lbh-body-s">
+            <PropertyFlags
+              canRaiseRepair={canRaiseRepair}
+              tenure={tenure}
+              propertyReference={propertyReference}
+            />
+          </div>
+          <h2 className="lbh-heading-h2 govuk-!-margin-top-6">
+            Work order task details
+          </h2>
+          <form
+            role="form"
+            id="repair-request-form"
+            onSubmit={handleSubmit(onSubmit)}
+          >
+            {canAssignFollowOnRelationship(user) &&
+              NEXT_PUBLIC_RELATED_WORKORDRES_TAB_ENABLED === 'true' && (
+                <RaiseWorkOrderFollowOn
+                  register={register}
+                  errors={errors}
+                  propertyReference={propertyReference}
+                  watch={watch}
+                />
+              )}
+
+            <RepairsFinderInput
+              propertyReference={propertyReference}
+              register={register}
+              setTotalCost={setTotalCost}
+              setContractorReference={setContractorReference}
+              setTradeCode={setTradeCode}
+            />
```

Comment:
Essentially the same form, except it includes the new Repairs finder fields, and removes contractor, trade, sorcode fields